packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-keys.js

```diff
@@ -0,0 +1,41 @@
+const loginEnBundle = require('../../../i18n/src/json/login.json');
+const loginokPLBundle = require('../../../i18n/src/json/login_ok_PL.json');
+const _ =  require('underscore');
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect missing keys from login_ok_PL.properties',
+      category: 'i18n Issues',
+      recommended: true,
+    },
+    messages: {
+      noMissingKeys: 'login.properties ( {{noOfKeysLoginEn}}: keys ) and login_ok_PL.properties ( {{noOfKeysLoginokPL}}: keys), are out of sync. Missing keys from login_ok_PL.properties. Run grunt exec:pseudo-loc to re-generate login_ok_PL.properties.'
+    },
+  },
+  create(context) {
+    return {
+      'Program': function (node) {
+        const fileName = context.getFilename();
+        if (fileName.includes('login.json')) {
```

Comment:
Enhancement: Allow this file to be configurable so we can port this over to Courage, Enduser, etc. in the future.

Reply:
Since courage, enduser are internal packages we have it integrated with the build so the files should be in sync.